src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -53,6 +53,9 @@ const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
     } else {
       try {
         await requestCRReview({ userIds: reviewerIds });
+        reviewerIds.forEach(function (value) {
```

Comment:
you should do this like `(userId) => toast.success(...)`, it makes it clearer what variable you're using and arrow functions are a bit cleaner

Reply:
also you should use the userid to retrieve the name of the user of the list of users we have fetched above

Reply:
Got it. Trying that